components/unified-header/main-navigation.tsx

```
@@ -0,0 +1,134 @@
+import Link from "next/link"
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu"
+import { ChevronDown } from "lucide-react"
+import styles from "./main-navigation.module.css"
+import { NavigationLink } from "./index"
+
+import { NavigationLink } from "./index"
```

Biome (1.9.4)

[error] Shouldn't redeclare 'NavigationLink'. Consider to delete it or rename it.

'NavigationLink' is defined here:


(lint/suspicious/noRedeclare)